packages/@aws-cdk/aws-rds/lib/private/util.ts

```diff
@@ -3,6 +3,8 @@ import * as s3 from '@aws-cdk/aws-s3';
 import { Construct, CfnDeletionPolicy, CfnResource, RemovalPolicy } from '@aws-cdk/core';
 import { IEngine } from '../engine';
 
+export const DEFAULT_PASSWORD_EXCLUDE_CHARS = " %+~`#$&*()|[]{}:;<>?!'/@\"\\";
```

Comment:
probably worth a comment to include the rationale (i.e. from the commit body, these are problematic with the shell / DMS) in case we ever want to loosen. Or how we selected these characters... something to provide more of a clue :)

Reply:
Done!